README-ja.md

```diff
@@ -116,42 +116,42 @@ const signature = wallet.sign(message);
 wallet.verify(message, signature); // true
 ```
 
-### XpringClient
+### XRPClient
 
-「XpringClient」はXRPレジャーへのゲートウェイです。 「XpringClient」は、リモートアダプターのURLである単一のパラメーターで初期化されます（上記の「サーバー側コンポーネント」セクションを参照）。
+「XRPClient」はXRPレジャーへのゲートウェイです。 「XRPClient」は、リモートアダプターのURLである単一のパラメーターで初期化されます（上記の「サーバー側コンポーネント」セクションを参照）。
 
 ```javascript
-const { XpringClient } = require("xpring-js");
+const { XRPClient } = require("xpring-js");
 
 const remoteURL = "grpc.xpring.tech:80";
-const xpringClient = XpringClient.xpringClientWithEndpoint(remoteURL);
+const xrpClient = XRPClient.xrpClientWithEndpoint(remoteURL);
```

Comment:
We should probably update these code snippets to be on par with the english version. Can you either file an asana talk or update them?

Reply:
Yes working on this in separate PR now